app/locale/vi.coffee

```diff
@@ -678,14 +678,14 @@ module.exports = nativeDescription: "Tiếng Việt", englishDescription: "Vietn
 #    combo_not_all_concepts_used: "You used __complete__ out of the __total__ concepts possible to solve the challenge. Try to get all __total__ concepts next time!"
 #    start_challenge: "Start Challenge"
     next_game: "Game kế tiếp"
-#    languages: "Languages"
-#    programming_language: "Programming language"
     show_menu: "Hiện game menu"
     home: "Home" # Not used any more, will be removed soon.
     level: "Level" # Like "Level: Dungeons of Kithgard"
     skip: "Bỏ qua"
-    game_menu: "Game menu"
     restart: "Khởi động lại"
+    languages: "Ngôn ngữ"
```

Comment:
move languages and programming_language translation to line 681 and 682 as before